modules/ROOT/pages/find-and-fix-conformance-issues.adoc

```diff
@@ -1,34 +1,32 @@
 = Finding and Fixing Conformance Issues
 :page-aliases: view-conformance-status-in-exchange.adoc
 
-To find and fix API conformance:
+As you govern and monitor APIs for conformance, you can continually improve conformance by empowering developers and implementors to find and fix conformance issues.
 
-. Initiate validation of APIs against rulesets in:
-+
-* xref:create-profiles.adoc[API Governance] for all supported API aspects
-* xref:find-conformance-issues.adoc[Design Center] for  specifications only
-* xref:find-conformance-issues.adoc#validate-in-cli[Anypoint CLI] for specifications only
-+
-. View conformance information as follows: 
-+
-* For an API validated using active profiles in API Governance, view conformance information in the:
-+
-** <<governance-validation-report,Validation report in API Governance>> for all supported API aspects
-** <<view-conformance-in-exchange,Conformance Status page for APIs in Exchange>> for specifications only
-** <<view-governed-apis-for-rulesets,Governed APIs page for rulesets in Exchange>> for specifications only
-+ 
-* To view conformance information for an API regardless of whether it's governed by an active profile, you can use:
-+
-** xref:find-conformance-issues.adoc#project-errors[Design Center] for specifications only
-** xref:find-conformance-issues.adoc#validate-in-cli[Reports generated through CLI] for specifications only
-+
-. Fix the issues using:
-+
-** xref:find-conformance-issues.adoc#project-errors[Design Center] for specifications
+Governance admins typically:
+
+* Initiate validation of APIs against rulesets in xref:create-profiles.adoc[API Governance] for all supported API aspects
+* View conformance information in the <<governance-validation-report,validation report in API Governance>>
+* Notify developers or implementors about conformance issues
+
+Developers and implementors typically:
+
+* Initiate validation of API specifications against rulesets in xref:find-conformance-issues.adoc[Anypoint Code Builder or Design Center].
+* View conformance information for each rule in the context of the open specification project. 
+
+* Fix conformance issues using the following:
+
+** xref:find-conformance-issues.adoc#fix-conformance-issues[Anypoint Code Builder or Design Center] for specifications
 ** xref:fix-documentation-conformance-issues.adoc[Exchange] for documentation
 ** xref:fix-catalog-info-conformance-issues.adoc[Exchange] for catalog information
 ** xref:fix-instance-conformance-issues.adoc[API Manager] for instances
 
+Regardless of which tasks you're responsible for in governance for your organization, you can view conformance information in several places in Anypoint Platform for APIs that are centrally-governed by API Governance:
+
+* In the <<governance-validation-report,validation report in API Governance or API Manager>> 
```

Comment:
Suggest removing "In the" (since it's repeated and it makes it harder to scan for the meat).